ldb/static/partials/ldb/person.html

```diff
@@ -275,9 +275,9 @@
         <a href class="icon-refresh" ng-hide="editmode" ng-click="person.student_model.confirm()"></a>
       </div>
 
-      <div ng-show="person.student_model.graduated && !editmode"><strong>Afgestudeerd</strong></div>
+        <div ng-show="person.student_model.enroled && !editmode"><strong>Ingeschreven</strong></div>
```

Comment:
Ik vind het raar om in de standaard situatie iets dik gedrukt weer te geven en bij de afwijkende situatie niets neer te zetten hierover.